test/functional/test-types.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '../../src/types';
+
+describe('Types', () => {
+  describe('toArray', () => {
+
+    it('should return empty array if null passed', () => {
+      expect(types.toArray(null).length).to.equal(0);
+      expect(types.toArray(undefined).length).to.equal(0);
+    });
+
+    it('should convert NodeList to array', () => {
+      const parent = document.createElement('div');
+      parent.appendChild(document.createElement('p'));
+      parent.appendChild(document.createElement('span'));
+      parent.appendChild(document.createElement('div'));
+      const arr = types.toArray(parent.childNodes);
+      expect(arr[0]).to.equal(parent.childNodes[0]);
+      expect(arr.length).to.equal(3);
+      expect(Array.isArray(arr)).to.be.true;
+    });
+
+    it('should convert HTMLCollection to array', () => {
```

Comment:
It doesn't hurt it just adds more tests to different types of objects (above is `NodeList` and this is `HTMLCollection`.